pyproject.toml

```diff
@@ -50,6 +50,7 @@ Pillow = "10.3.0"
 numpy = "^1.26.4"
 fakeredis = "^2.21.3"
 yarl = "^1.9.4"
+django = "^5.0.6"
```

Comment:
This won't do, you'll need to find a way to do this without adding the entirety of django.